### DISCUSSION TITLE
[Unit Test Framework Package] how to run a single test item

### DISCUSSION THREAD

**Discussion by yanghansen65535** [tianocore/edk2#4366] [Q&A] [answered]
I'm trying to use UnitTestFrameworkPkg to organize my test cases.
Now I'm able to create several test items, build .efi file and execute it in UEFI shell successfully. When I execute .efi in UEFI shell, it could test all test items continuously and automatically.
Is there any way to select one specific case to run? Sometimes need to debug one specific case, rerun it more than one times.
thanks!

**Comment by spbrogan** [comment, marked as answer]
I am not aware of anyway to run only a subset or a selected UEFI shell tests without making source code modifications.  Seems like a pretty reasonable request so if you wanted to file a bugzilla request and/or implement a change that would be great.  For host-based testing we do have this capability given the native frameworks we use.
